cereal/car.capnp

```diff
@@ -198,6 +198,8 @@ struct CarState {
   steeringPressed @9 :Bool;        # if the user is using the steering wheel
   steerFaultTemporary @35 :Bool;   # temporary EPS fault
   steerFaultPermanent @36 :Bool;   # permanent EPS fault
+  lkasDisabled @55 :Bool;          # LKAS being enabled is required to control car
+  lkasEnabled @56 :Bool;           # LKAS being disabled is required to control car
```

Comment:
this isn't a great name... there must be a better way to describe these here

Reply:
I have a suspicion one or the other can be removed now, so just going to make one alert cover this